tests/test-buffer-parser.cpp

```diff
@@ -223,6 +223,168 @@ TEST_CASE("Use a buffer parser without capture groups", "[BufferParser]") {
     parse_and_validate(buffer_parser, cInput, {expected_event});
 }
 
+/**
+ * @ingroup test_buffer_parser_capture
+ *
+ * @brief Tests the buffer parser behavior when using capture groups in variable schemas.
```

Comment:
Just to make it consistent with the other test.
```suggestion
 * @brief Validates tokenization behavior when using capture groups in variable schemas.
```